bin/run

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env node
+
+/* eslint-disable @typescript-eslint/no-var-requires */
+const fs = require('fs')
+const path = require('path')
+const project = path.join(__dirname, '../tsconfig.json')
```

Comment:
I have to investigate this. I found this file somewhere online and it worked so I was happy.

Reply:
I don't see `tsconfig.json` in the output dir so seems we can use that.

Thanks for the pointer. It's hard to get more docs about this `oclif-dev` package. I don't see any benefit from using that or a manifest file for now. Tried it and there was no change. Also couldn't find a hint in `bbk-devtools`.

We can use this tool to pack the CLI tool into releases for different platforms though.

Reply:
ok, it's not a blocker anyways. would just like to get to the bottom of it because i vaguely remember Barath explaining this to me once.

@barathvk do you remember why we needed the `oclif.manifest.json` file in `bbk-devtools`? i also can't seem to find any clear documentation on the oclif pages around it